packages/help-center/src/components/back-button.tsx

```diff
@@ -35,22 +50,13 @@ export const BackButton = ( { onClick, backToRoot = false, className }: BackButt
 	}
 
 	return (
-		<Button className={ buttonClassName } onClick={ onClick || defaultOnClick }>
-			<Icon icon={ chevronLeft } size={ 18 } />
-			{ __( 'Back', __i18n_text_domain__ ) }
-		</Button>
-	);
-};
-
-export const BackButtonHeader = ( { children, className }: BackButtonProps ) => {
-	return (
-		<div className={ clsx( 'help-center-back-button__header', className ) }>
-			<Flex justify="space-between">
-				<FlexItem>
-					<BackButton />
-				</FlexItem>
-				{ children }
-			</Flex>
-		</div>
+		<span className={ buttonClassName }>
+			<Icon
+				onClick={ onClick || defaultOnClick }
+				icon={ isHelpCenterHome ? <DragIcon /> : chevronLeft }
+				size={ 18 }
+			/>
+			{ buttonText ?? defaultButtonText }
```

Comment:
I think this could be slightly confusing since this is in the `BackButton` component but isn't actually part of the button. Also since the button only exists inside the header now we could probably get rid of the text all together and output the text inside the header component.